docs/contribution/nearcore.md

```diff
@@ -114,3 +114,108 @@ Official image is published at `nearprotocol/nearcore`
+real	4m42.485s user	28m47.223s sys	2m18.833s
+# touch */*/*/*.rs; time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER= cargo build -p neard --release
+real	0m51.620s user	4m13.180s sys	0m11.254s
+```
+
+# cache results from previous compilations using sccache
+```bash
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	4m1.409s user	5m48.943s sys	0m51.071s
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	1m14.038s user	5m25.992s sys	0m57.247s
+```
+
+## Recommended settings
+
+# Setting for building release binary without debug symbols
+```bash
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	4m1.409s user	5m48.943s sys	0m51.071s
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	1m14.038s user	5m25.992s sys	0m57.247s
+# touch */*/*/*.rs; time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	0m7.371s user	0m5.646s sys	0m1.763s
+
+```
+
+# Setting for building release binary with debug symbols and reduced inlining
+```bash
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6 -C inline-threshold=25 -C debuginfo=2' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	4m3.216s user	6m2.943s sys	0m54.196s
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6 -C inline-threshold=25 -C debuginfo=2' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	1m30.039s user	5m50.652s sys	1m2.509s
+# touch */*/*/*.rs; time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6 -C inline-threshold=25 -C debuginfo=2' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	0m8.564s user	0m5.975s sys	0m2.771s pmnoxx@ubuntu:~/repos
+```
+
+## Instalation guide
+
+# sccache
+Follow guide at https://github.com/mozilla/sccache to install sccache.
+If you are unable to install sccache you can remove `RUSTC_WRAPPER` option.
+
+#lld
+
+For ubuntu use:
+```
+sudo apt update
+sudo apt install lld
+```
+
+For other systems follow link at https://lld.llvm.org/
+If you can't install lld, you can remove option `-C link-arg=-fuse-ld=lld`.
+
+
+## Clion/Intellij with Rust plugin
+Currently Rust plugin doesn't support passing enviroment variables to cargo.
+Your best option is to either modify Cargo.toml or replace
+`/home/pmnoxx/.cargo/bin/cargo` with a bash script, which sets proper
```

Comment:
probably make sense to hide your username?

Reply:
fixed